src/app/badges/user-info/user-info.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="user__wrapper">
+  <div class="button-wrapper">
+    <button class="btn block-button" (click)="dialogsService.openEditProfileDialog()">Edit</button>
+  </div>
+
+  <app-avatar [user]="user" [size]="'big'"></app-avatar>
+
+  <h3 class="user__name">{{ user.firstName + ' ' + user.lastName }}</h3>
```

Comment:
provide class names in kebab-case. double check all your class names in whole application and fix them.